ph-child.php

```diff
@@ -29,204 +30,202 @@
  */
 
 // Plugin Folder Path.
-if (!defined('PH_CHILD_PLUGIN_DIR')) {
-	define('PH_CHILD_PLUGIN_DIR', plugin_dir_path(__FILE__));
+if ( ! defined( 'PH_CHILD_PLUGIN_DIR' ) ) {
+	define( 'PH_CHILD_PLUGIN_DIR', plugin_dir_path( __FILE__ ) );
 }
 
 // Plugin Folder URL.
-if (!defined('PH_CHILD_PLUGIN_URL')) {
-	define('PH_CHILD_PLUGIN_URL', plugin_dir_url(__FILE__));
+if ( ! defined( 'PH_CHILD_PLUGIN_URL' ) ) {
+	define( 'PH_CHILD_PLUGIN_URL', plugin_dir_url( __FILE__ ) );
 }
 
 // Plugin Root File.
-if (!defined('PH_CHILD_PLUGIN_FILE')) {
-	define('PH_CHILD_PLUGIN_FILE', __FILE__);
+if ( ! defined( 'PH_CHILD_PLUGIN_FILE' ) ) {
+	define( 'PH_CHILD_PLUGIN_FILE', __FILE__ );
 }
 
 // include child functions
 require_once 'ph-child-functions.php';
 
-if (!class_exists('PH_Child')) :
+if ( ! class_exists( 'PH_Child' ) ) :
 	/**
 	 * Main PH_Child Class
 	 * Uses singleton design pattern
 	 *
 	 * @since 1.0.0
 	 */
-	final class PH_Child
-	{
+	final class PH_Child {
+
 		/**
 		 * Make sure to whitelist our option names
 		 *
 		 * @var array
 		 */
-		protected $whitelist_option_names = array();
+		protected $whitelist_option_names = [];
 
 		/**
 		 * Get things going
 		 */
-		public function __construct()
-		{
-			if (defined('PH_VERSION')) {
-				add_action('admin_notices', array($this, 'parent_plugin_activated_error_notice'));
+		public function __construct() {
+			if ( defined( 'PH_VERSION' ) ) {
+				add_action( 'admin_notices', [ $this, 'parent_plugin_activated_error_notice' ] );
 				return;
 			}
 
-			$this->whitelist_option_names = array(
-				'ph_child_id'           => array(
-					'description'       => __('Website project ID.', 'project-huddle'),
+			$this->whitelist_option_names = [
```

Comment:
Let's not change to PHP short arrays right away, This is only PHP 5.6+. 
Not sure if currently plugin explicitly mentions if it is PHP 5.6+

If we want to make this change, we will have to do this in two parts, first raise php requirement to 5.6 and then after some time and 1-2 updates, start using PHP 5.6+ features

Reply:
Correction, it's php 5.4+

Reply:
Actually, looks like the plugin does mention it is PHP 5.6+ - https://wordpress.org/plugins/projecthuddle-child-site/

Reply:
Yes, should we keep these changes or lets wait for upcoming releases?

Reply:
@sushmak02 - Can you revert array syntax change for now? you can do this easily by adding the phpcs rule which comes with a autofixer.

I think you just need to change `Generic.Arrays.DisallowLongArraySyntax.Found` to `Generic.Arrays.DisallowShortArraySyntax.Found`